tests/utils/db.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+var PouchDB = process.browser ? global.PouchDB : require('pouchdb')
+var uuid = PouchDB.utils.uuid
+
+module.exports = function (name) {
+  name = name || uuid(10)
+
+  var options = process.browser ? {
+    adapter: 'memory'
+  } : {
+    db: require('memdown')
+  }
+
+  return new PouchDB(name, options)
+}
```

Comment:
instead of using `uuid`, can you implement it as we did it in `pouchdb-hoodie-api`:
https://github.com/hoodiehq/pouchdb-hoodie-api/blob/master/tests/utils/db.js